### ISSUE TITLE
2x2 crafting does not work in creative

### ISSUE THREAD

**Issue by Camotoy** [GeyserMC/Geyser#404]
**Describe the bug**
Attempting to put an item in the 2x2 crafting slot does not work.

**To Reproduce**
- Be in creative
- Try to put an item into the 2x2 crafting slot

**Server Version**
Paper-184

**Geyser Version**
Master 71

**Minecraft: Bedrock Edition Version**
v1.14.60

**Comment by DoctorMacc** [comment]
Java Edition no longer has a crafting table in the creative inventory - we cannot fix this.

**Comment by DoctorMacc** [comment]
Actually, we should consider sending a one-time message to let users know you can't do that.

**Comment by Heath123** [comment]
> Java Edition no longer has a crafting table in the creative inventory - we cannot fix this.

It's possible because the client can create whatever items it wants in creative, and the server sends recepies. It's probably not worth implementing for now though (maybe this should be tagged Low Priority instead of Won't Fix?)

**Comment by DoctorMacc** [comment]
Addressed with #422.

**Comment by Heath123** [comment]
This is possible to actually fix though

**Comment by DoctorMacc** [comment]
I'm closing this since we've addressed it double-time. I think this is possible to properly implement, but to be on the safe side, so we don't raise any red flags, it should probably be a non-standalone thing only.

**Comment by Heath123** [comment]
Red flags?

**Comment by DoctorMacc** [comment]
Right, the inventory and creative menu are the same thing.

So then the only issue would be if you craft something that isn't available in the creative inventory.

I'll reopen this until we all come to a solid conclusion.

**Comment by MrBlockCat** [comment]
Is this still considered an issue after #426 and #422 ? I personally think those look like a good implementation of a fix for this, though I haven't gotten to try them out personally.

**Comment by RednedEpic** [comment]
Closing - moving to the Current Limitations page on the wiki.
